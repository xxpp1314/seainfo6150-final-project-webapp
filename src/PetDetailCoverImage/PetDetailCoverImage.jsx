import React from 'react';
import PropTypes from "prop-types";
import styles from "./PetDetailCoverImage.module.css";

const PetDetailCoverImage = props => {
    return (
        <img className={styles.image} src={props.url} alt=""/>
    )
}

PetDetailCoverImage.propTypes = {
    url: PropTypes.string.isRequired,
};

export default PetDetailCoverImage;
