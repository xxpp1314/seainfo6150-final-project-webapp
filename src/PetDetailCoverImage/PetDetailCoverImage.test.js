import React from "react";
import { render } from "@testing-library/react";
import PetDetailCoverImage from "./PetDetailCoverImage.jsx";

const imageurl="https://dl5zpyw5k3jeb.cloudfront.net/photos/pets/50829956/1/?bust=1619138292&width=1080";

it("renders the PetDetailCoverImage correctly", () => {
  const { container } = render(
        <PetDetailCoverImage
            url={imageurl}
        />
    );
  expect(container).toMatchSnapshot();
});