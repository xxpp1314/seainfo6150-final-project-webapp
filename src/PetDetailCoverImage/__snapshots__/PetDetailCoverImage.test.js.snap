// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PetDetailCoverImage correctly 1`] = `
<div>
  <img
    alt=""
    class="image"
    src="https://dl5zpyw5k3jeb.cloudfront.net/photos/pets/50829956/1/?bust=1619138292&width=1080"
  />
</div>
`;
